lang/cs/Org.Apache.REEF.Wake/Remote/IConnectionRetryHandler.cs

```diff
@@ -15,7 +15,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#if NOT_DOTNET_BUILD
```

Comment:
There a no constants for the new build, but I should be able them and flip this around.